components/dashboard/src/Menu.tsx

```diff
@@ -232,13 +228,13 @@ export default function Menu() {
         ) {
             teamSettingsList.push({
                 title: "Usage",
-                link: `/t/${team.slug}/usage`,
+                link: `/org-usage`,
```

Comment:
we can align usage and org-usage in follow-up steps, same for billing/org-billing

Reply:
Sounds good - the `/org-usage` path here felt weird to me since nothing else is prefixed by `/org-`. Do you mean it'll be easier to change after this PR to something like that?

Reply:
It's just fewer changes. the PR is already quite big.